app/dao/permissions_dao.py

```diff
@@ -7,5 +16,30 @@ class PermissionDAO(DAOClass):
     class Meta:
         model = Permission
 
+    def get_query(self, filter_by_dict={}):
```

Comment:
Would we query permissions by anything other than the user? Is there a user.persmissions mapping, so when you get a user you get the permisssions / save the user update the permissions?